### PR TITLE
add GitHub issue syncing example

### PR DESCRIPTION
**Pull request by malomarrec**

### CODE REVIEW

File: github-issues/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Container image that runs your code
+FROM python:3.6-rc-alpine
+
+RUN pip install requests
+
+# Copies your code file from your action repository to the filesystem path `/` of the container
+COPY sync-issue.py /sync-issue.py
+
+# Code file to execute when the docker container starts up (`entrypoint.sh`)
+  ENTRYPOINT ["python","sync-issue.py"]
```
Comment (mrnugget) [inline github-issues/Dockerfile:10]: What's with that whitespace here? You trying to give me a heartattack?

File: github-issues/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+**Note**: by convention, all the issues tracked by this script must be named `batch-change/<batch-change-name>`. In this example `batch-change/refactor-go-gh-issue`
+
+## How to
+- Modify the `gh_username` flag with your own in `gh-issues.batch.yml`
+- Create a [GitHub Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token) and set it as the `GH_TOKEN` environment variable.
+
+```bash
+GH_TOKEN=mysecrettoken
+src campaign apply -f gh.issues.batch.yml
```
Comment (mrnugget) [inline github-issues/README.md:24]:
```suggestion
export GH_TOKEN=mysecrettoken
src campaign apply -f gh.issues.batch.yml
```

File: github-issues/gh-issues.batch.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+name: refactor-go-gh-issue
+description: |
+  This campaign creates Jira tickets for each repository where fmt.Sprintf is replaced by strconv.Itoa.
+  It also keeps the tickets updated when the campaigns is re-applied.
+on:
+  - repository: github.com/sourcegraph-testing/tiny-go-test-repository
+
+steps:
+  - run: comby -in-place 'fmt.Sprintf("%d", :[v])' 'strconv.Itoa(:[v])' .go -matcher .go -exclude-dir .,vendor; echo "issue"
```
Comment (mrnugget) [inline github-issues/gh-issues.batch.yml:9]:
```suggestion
  - run: comby -in-place 'fmt.Sprintf("%d", :[v])' 'strconv.Itoa(:[v])' ${{ join repository.search_result_paths " " }}
```

If you use templating here you don't need the `exclude-dir` stuff etc.

**Review by mrnugget — APPROVED** [review]
It would be cool if these were idempotent, where they would check whether a issue for the given campaign and repo/workspace already exists on GitHub and if yes only update it.

**Comment by malomarrec** [comment]
@mrnugget they **are** idempotent!

**Pull request merged**